File: class_member/2.rb
class Cs
  #class method는 class 소속
  def Cs.class_method
    p "Class method"
  end
  #instance method는 instance 소속
  def instance_method
    p "Instance method"
  end
end
i=Cs.new
Cs.class_method
i.instance_method

File: class_member/2.py
class Cs:
    #static & class method 는 class 소속
    #장식자
    @staticmethod
    def static_method():
        print("Static method")
    @classmethod
    def class_method(cls):
        print("class method")
    def instance_method(self):
        print("instance method")

i=Cs()
Cs.static_method()
Cs.class_method()
i.instance_method()
